tools/ci/testlist/arm-10.dat

```diff
@@ -4,7 +4,7 @@
 
 /arm/stm32/stm32f103-minimum,CONFIG_ARM_TOOLCHAIN_GNU_EABI
 
-/arm/stm32/stm32f334-disco,CONFIG_ARM_TOOLCHAIN_CLANG
+/arm/stm32/stm32f334-disco,CONFIG_ARM_TOOLCHAIN_GNU_EABI
```

Comment:
Let's wait for CI to complete. If all is fine then let's separate this particular change into a separate commit.

Reply:
If the issue is solved by optimization so let's revert this chnage

Reply:
Done 👍🏼

Reply:
I'll see what config fails and apply the same optimization to it